### DISCUSSION TITLE
why don't use a framework/lib like react or vue.

### DISCUSSION THREAD

**Discussion by rainbowatcher** [shi-yan/Epiphany#45] [General]
Personally, i think using some of them can makes the community more engaged.

**Comment by shi-yan** [comment]
Thank you!

Because 90% of the code is configuring prosemirror, which doesn't seem to go well or needs a framework.

Plus, I'm in the middle of switching my stack from VUE to solid. I'm not familiar with solid enough yet, so I'm skipping it for now. 

In the future, I might switch to Solid.
